src/Compactable.ts

```diff
@@ -119,51 +98,105 @@ export interface Compactable3C<F extends URIS3, U, L> {
  * @function
  * @since 1.7.0
  */
-export function compactDefault<F extends URIS3, U, L>(
```

Comment:
Why the renaming? Also I don't understand `compactDefaultFilterMap` and `separateDefaultPartitionMap`, they seem trivial. Maybe we can just add some documentation explaining how to derive compact/separate from filterMap/partitionMap and avoid some bloat

Reply:
Renaming is to be explicit in what is needed for default implementation since there's more than one way to get it. The name reflects purescript implementation, firstly I picked this convention: `get<Name>From<OtherName>` which is more readabe.

Although they are trivial you still need to write them each time, I think it's just handy to have them as a part of the library as well as they are a good way of reflecting relation between these typeclasses, don't you think?